File: src/gatling/java/com/example/javaModels/Comment.java
package com.example.javaModels;

import com.fasterxml.jackson.annotation.JsonIgnoreProperties;
import com.fasterxml.jackson.annotation.JsonInclude;
import lombok.*;
import lombok.experimental.Accessors;

import java.util.ArrayList;
import java.util.List;

@JsonIgnoreProperties({"id"})
@JsonInclude(JsonInclude.Include.NON_NULL)
@Data
@Builder(setterPrefix = "with")
@Accessors(chain = true)
@AllArgsConstructor
@NoArgsConstructor
public class Comment {

    private int id;
    private int post;
    private String name;
    private String email;
    private List<Integer> likes;
    private List<Integer> dislikes;
    private String body;
    private String createdAt;

}

File: src/gatling/kotlin/com/example/modelds/Comment.kt
package com.example.modelds

import com.fasterxml.jackson.annotation.JsonIgnoreProperties
import com.fasterxml.jackson.annotation.JsonInclude


@JsonIgnoreProperties("id")
@JsonInclude(JsonInclude.Include.NON_NULL)
data class Comment(
    private val id: Int? = null,
    private val post: Int,
    private val name: String,
    private val email: String,
    private val likes: MutableList<Int>,
    private val dislikes: MutableList<Int>,
    private val body: String,
    private val createdAt: String): Model
